### ISSUE TITLE
Upgrading 3.0.3 -> 3.2.0 with Webpack5, Yarn2, dexie.js is not exported from package

### ISSUE THREAD

**Issue by phil-w** [dfahlander/Dexie.js#1437]
Upgrading from 3.0.3 to 3.0.2 my webpack 5 / Yarn 2 build starts to fail with:
```
ERROR in ./ClientApp/.../dexieDB.js 3:0-26
Module not found: Error: Package path ./dist/dexie.js is not exported from package 
...Repo\.yarn\cache\dexie-npm-3.2.0-575507c806-59b6fd600a.zip\node_modules\dexie\ 
(see exports field in ... node_modules\dexie\package.json)
```
If I look up the differences between those two package.json files, the error is caused by the addition in 3.2.0 of an "exports", viz:
` "exports": "./dist/modern/dexie.mjs",`

The new "exports" entry explicitly exports "dexie.mjs", where as my webpack/ yarn2 stuff is apparently expecting dexie.js, as the error message suggests. My import has been: `import Dexie from 'dexie';`

The release notes: https://github.com/dfahlander/Dexie.js/releases don't mention this specifically.
[This ](https://github.com/dfahlander/Dexie.js/commit/beae520304ac205b8cb7112c56c58aa74295d710#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519)is the change wot broke me build.

I can probably alias it somehow, but maybe I'm doing something stupid? 

---
Update: yes, it's my stupidity:
I was aliasing to dexie.js, not dexie.mjs, so that'd cause this. Please close!

**Comment by dfahlander** [comment]
Ok thanks for finding the reason. Closing

**Comment by dfahlander** [comment]
See #1439 which is resolved in 3.2.1-beta.2
